cookbook/specs/axelar.json

```diff
@@ -1682,6 +1700,24 @@
                                     "hanging_api": true
                                 },
                                 "extra_compute_units": 0
+                            },
+                            {
+                                "name": "/axelar/reward/v1beta1/inflation_rate/{validator}",
```

Comment:
this api is not supported as a post, only as a get:
service QueryService {
  rpc InflationRate ( .axelar.reward.v1beta1.InflationRateRequest ) returns ( .axelar.reward.v1beta1.InflationRateResponse ) {
    option (.google.api.http) = {
      get: "/axelar/reward/v1beta1/inflation_rate/{validator}"
      additional_bindings: <
        get: "/axelar/reward/v1beta1/inflation_rate"
      >
    };
  }

Reply:
Done. I'll have to fix the update script as well in a different PR.